logstash/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
@slim-bean do we need to have a license ?

Reply:
No separate license, this would fall under the license for the project itself

Reply:
Will remove